engine/session.js

```diff
@@ -40,6 +40,9 @@ class Session {
     this._events = [];
     this.averageSegmentDuration = AVERAGE_SEGMENT_DURATION;
     this.use_demuxed_audio = false;
+    this.use_vtt_Subtitles = false;
```

Comment:
Why the capital `S` here? Would be more consistent to call it `use_vtt_subtitles`

Reply:
fixed